dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/WorkerGroupService.java

```diff
@@ -159,13 +159,19 @@
         for (String workerGroup : workerGroupList) {
             String workerGroupPath = workerPath + "/" + workerGroup;
             List<String> childrenNodes = zookeeperCachedOperator.getChildrenKeys(workerGroupPath);
+            String timeStamp = "";
+            for (int i = 0; i < childrenNodes.size(); i++) {
+                String ip = childrenNodes.get(i);
+                childrenNodes.set(i, ip.substring(0, ip.lastIndexOf(":")));
+                timeStamp = ip.substring(ip.lastIndexOf(":"));
+            }
             if (CollectionUtils.isNotEmpty(childrenNodes)) {
                 availableWorkerGroupList.add(workerGroup);
```

Comment:
delete it

Reply:
> delete it
I don't know what you mean.
Delete which？